internal/service/fleet_test.go

```diff
@@ -82,12 +86,18 @@ func testFleetPatch(require *require.Assertions, patch api.PatchRequest) (*api.F
 			},
 		},
 	}
-	serviceHandler := ServiceHandler{
+	serviceHandler := &ServiceHandler{
 		store:           &FleetStore{FleetVal: fleet},
 		callbackManager: dummyCallbackManager(),
 	}
 	resp, status := serviceHandler.PatchFleet(context.Background(), "foo", patch)
 	require.NotEqual(int32(500), status.Code)
+	event, _ := serviceHandler.store.Event().List(context.Background(), uuid.New(), store.ListParams{})
```

Comment:
Always check errors even in tests
```suggestion
	event, err := serviceHandler.store.Event().List(context.Background(), uuid.New(), store.ListParams{})
	require.NoError(err)
```

Reply:
done